src/sentry/api/endpoints/organization_details.py

```diff
@@ -380,6 +393,16 @@ def save(self):
             org.name = data["name"]
         if "slug" in data:
             org.slug = data["slug"]
+        if "providerName" in data and "providerConfig" in data:
```

Comment:
@schew2381 should we always require `providerConfig`?

Reply:
I talked with @leedongwei about this and we agreed that they both have to be passed in together